Jenkinsfile_CNP

```diff
@@ -95,6 +95,7 @@ env.BEFTA_RESPONSE_HEADER_CHECK_POLICY="JUST_WARN"
 env.ELASTIC_SEARCH_FTA_ENABLED = "false"
 env.PACT_BROKER_FULL_URL = 'https://pact-broker.platform.hmcts.net'
 env.DEFAULT_COLLECTION_ASSERTION_MODE="UNORDERED"
+env.MCA_API_BASE_URL = "http://aac-manage-case-assignment-pr-294.service.core-compute-preview.internal".toLowerCase()
```

Comment:
why do we need this?

Reply:
This will not be checked in

The data-store PR was caching the callback URLs that were pointing to the AAT version of MCA rather than the PR-294 version

This points datastore casllback URLs to PR-294 version of MCA